sdk/monitor/monitor-opentelemetry/samples-dev/customMetric.ts

```diff
@@ -7,26 +7,30 @@
 
 import {
   AzureMonitorOpenTelemetryClient,
-  AzureMonitorOpenTelemetryConfig,
+  AzureMonitorOpenTelemetryOptions,
 } from "@azure/monitor-opentelemetry";
 
 // Load the .env file if it exists
 import * as dotenv from "dotenv";
 dotenv.config();
 
-let config = new AzureMonitorOpenTelemetryConfig();
+const config: AzureMonitorOpenTelemetryOptions = {
+  azureMonitorExporterConfig: {
+    connectionString: "<YOUR_CONNECTION_STRING>",
```

Comment:
The sample should try and read the connection string in from the environment such as `process.env.AZURE_MONITOR_CONNECTION_STRING`